docs/blog.md

```diff
@@ -404,11 +404,11 @@ Now we had our backend polling logic and waku setup ready, lets import these fun
     }
 ```
 
-You will notice the code blocks above uses other state values like wakuNodeStatus, lastSynced these are metrics to show user the peer connection status and last received time of a waku message.
+You will notice the code blocks above uses other state values like `wakuNodeStatus` or `lastSynced` these are metrics to show the user the peer connection status and last time we received a message ove Waku.
 
 #### Wrap
 
-Although the problem that I solved here can be implmented many ways, having decentralized solutions makes your app fully decentralized. Waku can be applied in several other cases too.
+Although the problem that I solved here can be implmented in many ways, having decentralized solutions makes your app fully decentralized. Waku can be applied in several other cases too.
```

Comment:
Would you mind talking a bit more about Waku here? Maybe explain the decentralization and how Waku helps with it?